### PR TITLE
Fix continuous assignment on variable declaration

### PR DESCRIPTION
**Pull request by rkapuscik**

### CODE REVIEW

File: src/UhdmAst.cpp [hunk 1]
```diff
@@ -3220,20 +3245,25 @@ AstNode* visit_object(vpiHandle obj_h, UhdmShared& shared) {
...
         // Dummy statement to keep parsing
         return new AstTime(make_fileline(obj_h),
                            VTimescale::TS_1PS);  // TODO: revisit once we have it in UHDM
         break;
-    case vpiUnsupportedExpr:
-        v3info("\t! This expression is unsupported in UHDM: " << file_name << ":" << currentLine);
+    }
+    case vpiUnsupportedExpr: {
+        auto* fl = make_fileline(obj_h);
+        fl->v3info("\t! This expression is unsupported in UHDM: " << file_name << ":" << currentLine);
```
Comment (RRozak) [inline src/UhdmAst.cpp]: I would mention that the type can also be undefined. Surelog uses this node if the type is from package which is not included.

File: src/UhdmAst.cpp [hunk 2]
```diff
@@ -3220,20 +3245,25 @@ AstNode* visit_object(vpiHandle obj_h, UhdmShared& shared) {
         });
         return assert;
     }
-    case vpiUnsupportedStmt:
-        v3info("\t! This statement is unsupported in UHDM: " << file_name << ":" << currentLine);
+    case vpiUnsupportedStmt: {
+        auto* fl = make_fileline(obj_h);
+        fl->v3info("\t! This statement is unsupported in UHDM: " << file_name << ":" << currentLine);
         // Dummy statement to keep parsing
         return new AstTime(make_fileline(obj_h),
```
Comment (RRozak) [inline src/UhdmAst.cpp]: In other places we throw an error. If we print only information, it is possible that we won't notice it.

**Review by RRozak — COMMENTED** [review]
The problems that I found are not related to this PR, but I think it would be good idea to fix them.

File: src/UhdmAst.cpp (hunk 1 above)
Comment (rkapuscik) [reply src/UhdmAst.cpp]: Ideally that case would be covered by an `unsupported_typespec`, if it happens here we can't really fix it in the frontend and this should be fixed in Surelog.

I added a comment to help debugging that, but I'd suggest keeping the error message about the node itself.

File: src/UhdmAst.cpp (hunk 2 above)
Comment (rkapuscik) [reply src/UhdmAst.cpp]: Fixed.

**Review by RRozak — APPROVED** [review]

**Pull request merged**